ansible/doc/README.testbed.VsSetup.md

```diff
@@ -0,0 +1,123 @@
+# Testbed Setup
```

Comment:
Hi @lguohan . Is there a list of tests which are supported on this virtual testbed?